graphkb/match.py

```diff
@@ -538,6 +524,20 @@ def match_positional_variant(
             ),
         )
 
+    # disambiguate the variant type
+    variant_types_details = get_equivalent_terms(
+        conn,
+        parsed['type'],
+        root_exclude_term='mutation' if secondary_features else '',
+        ignore_cache=ignore_cache,
+    )
+
+    types = convert_to_rid_list(variant_types_details)
+
+    if not types:
+        variant_type = parsed['type']
+        raise ValueError(f'unable to find the term/category ({variant_type}) or any equivalent')
+
     matches.extend(
         conn.query(
             {
```

Comment:
Hi @mathieulemieux 

Should the function def cat_variant_query, below, also be using 'returnProperties': POS_VARIANT_RETURN_PROPERTIES, 
instead of ''returnProperties': VARIANT_RETURN_PROPERTIES',
?